portal/views/home_new.py

```diff
@@ -0,0 +1,179 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# ADDITIONAL TERMS – Section 7 GNU General Public Licence
+#
+# This licence does not grant any right, title or interest in any “Ocado” logos,
+# trade names or the trademark “Ocado” or any other trademarks or domain names
+# owned by Ocado Innovation Limited or the Ocado group of companies or any other
+# distinctive brand features of “Ocado” as may be secured from time to time. You
+# must not distribute any modification of this program using the trademark
+# “Ocado” or claim any affiliation or association with Ocado or its employees.
+#
+# You are not authorised to use the name Ocado (or any of its trade names) or
+# the names of any author or contributor in advertising or for publicity purposes
+# pertaining to the distribution of this program, without the prior written
+# authorisation of Ocado.
+#
+# Any propagation, distribution or conveyance of this program must include this
+# copyright notice and these terms. You must not misrepresent the origins of this
+# program; modified versions of the program must be marked as such and not
+# identified as the original program.
+from functools import partial
+
+from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from django.core.urlresolvers import reverse, reverse_lazy
+from django.contrib import messages as messages
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.forms import AuthenticationForm
+from recaptcha import RecaptchaClient
+from django_recaptcha_field import create_form_subclass_with_recaptcha
+
+from portal.forms.teach import TeacherLoginForm
+from portal.helpers.emails import send_verification_email, is_verified
+from portal.utils import using_two_factor
+from portal import app_settings
+from ratelimit.decorators import ratelimit
+
+recaptcha_client = RecaptchaClient(app_settings.RECAPTCHA_PRIVATE_KEY, app_settings.RECAPTCHA_PUBLIC_KEY)
+
+
+def teach_email_labeller(request):
+    if request.method == 'POST' and 'login' in request.POST:
+        return request.POST['login-email']
+
+    return ''
+
+
+def login_popup(request):
+    return render_forms(request, 'redesign/login_popup.html')
+
+
+def home_new(request):
+    return render_forms(request, 'redesign/home_new.html')
+
+
+def teach_new(request):
+    return render_forms(request, 'redesign/teach_new.html')
+
+
+def play_new(request):
+    return render_forms(request, 'redesign/play_new.html')
+
+
+def about_new(request):
+    return render_forms(request, 'redesign/about_new.html')
+
+
+def help_and_support_new(request):
+    return render_forms(request, 'redesign/help-and-support_new.html')
+
+
+def terms_new(request):
+    return render_forms(request, 'redesign/terms_new.html')
```

Comment:
We shouldn't need to worry about this anymore as the login popup and register popup will have their own page so they can have their own URL, this way the render_forms will only be called on one specific page.